hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeAdminDefaultMonitor.java

```diff
@@ -201,6 +201,7 @@ private void check() {
         iterkey).iterator();
     final List<DatanodeDescriptor> toRemove = new ArrayList<>();
     final List<DatanodeDescriptor> unhealthyDns = new ArrayList<>();
+    boolean inValidState = false;
 
     while (it.hasNext() && !exceededNumBlocksPerCheck() && namesystem
```

Comment:
Single Line?

Reply:
What do you mean by Single Line?

Reply:
`while (it.hasNext() && !exceededNumBlocksPerCheck() && namesystem.isRunning())`

Reply:
This change hasnt made my me. For fixing any such issues - new JIRA can be created to clean or modify such issues on module level.